refactor_me.py

```diff
@@ -3,24 +3,35 @@
 from collections import namedtuple, defaultdict
 from operator import itemgetter
 
-Expense = namedtuple('Expense', ('type_', 'amount'))
+Expense = namedtuple('Expense', ('category', 'amount'))
 
 
-def sum_expenses(expenses, min_amount=0):
-    aggregated_expenses = defaultdict(int)
-    for expense in expenses:
-        amount = expense.amount
-        if amount >= min_amount:
-            aggregated_expenses[expense.type_] += amount
-    return aggregated_expenses
+class BaseExpencesManager:
+    _buffer = ()
 
+    def add(self, expenses):
+        for expense in expenses:
+            self._buffer += (expense, )
 
-def print_expenses(expenses):
-    for expense, amount in sorted(expenses.items(), key=itemgetter(1)):
-        print(expense, amount)
+    def by_category(self, treshold=0):
```

Comment:
typo in `treshold`, also it is a good candidate for an instance level data attribute